data/hai/hai reveal news.txt

```diff
@@ -518,14 +518,10 @@ phrase "person you could suspect to be aliens"
 		"cat"
 
 
-########################################################################
-
-# Many different missions have the potential to trigger this so an event is used instead of missions in the "to show".
-event "news: hai video clip published"
 
 news "Hai video: human response"
 	to show
-		has "event: news: hai video clip published"
+		has "hr: secret leaks"
 		not "event: news: hai video clip debunked"
```

Comment:
```suggestion
		# not "event: news: hai video clip debunked"
```
You got rid of the event.

Reply:
This condition will do nothing anyway, Little use in commenting it out.